app/plugin_commands.go

```diff
@@ -18,7 +20,27 @@ type PluginCommand struct {
 
 func (a *App) RegisterPluginCommand(pluginId string, command *model.Command) error {
 	if command.Trigger == "" {
-		return fmt.Errorf("invalid command")
+		return errors.New("invalid command")
+	}
+	if command.AutocompleteData != nil && command.AutocompleteData.IsValid() != nil {
+		return errors.New("invalid autocomplete data in command")
```

Comment:
Why are you dropping the `error` returned from `IsValid()`? Mind just wrapping it?